src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/DefaultFindSearchInputOperation.java

```diff
@@ -51,19 +51,15 @@ private SearchInput toSearchInput(final SearchInputField in, final CaseType case
         final SearchInput result = new SearchInput();
         result.setLabel(in.getLabel());
         result.setOrder(in.getDisplayOrder());
+        CaseField caseField = caseType.getCaseField(in.getCaseFieldId());
```

Comment:
you can use the original `getCaseField` method that returns Optional here and throw exception at this level

Reply:
Changed as in above